scaper/audio.py

```diff
@@ -66,45 +27,40 @@ def get_integrated_lufs(filepath, min_duration=0.5):
 
     Parameters
     ----------
-    filepath : str
-        Path to audio file for computing LUFS
+    audio_array : np.ndarray
+        numpy array containing samples or path to audio file for computing LUFS
+    samplerate : int
+        Sample rate of audio, for computing duration
     min_duration : float
         Minimum required duration for computing LUFS value. Files shorter than
         this are self-concatenated until their duration reaches this value
         for the purpose of computing the integrated LUFS. Caution: if you set
         min_duration < 0.4, a constant LUFS value of -70.0 will be returned for
```

Comment:
Update to: if you set min_duration < 0.4, the LUFS estimate will be unstable.

OR... what DOES pyloudnorm do in this case?

Reply:
pyloudnorm actually errors out, so I add the tiling logic ahead of time. This path is tested via the dog bark regression file.

Reply:
Update docstring accordingly?